.github/workflows/integration-test.yml

```diff
@@ -0,0 +1,24 @@
+name: Integration Test
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Set up
+      env:
+        CLI_CONFIG: ${{ secrets.cliConfig }}
```

Comment:
1) It is stored in Github Secrets. It is json of cli_config file. The values on tokens technically does not matter since for smapi test we use mocked smapi and mocked lwa.

2) We don't run high level commands in git actions, since we did not decide what AWS account to use, so there is not need for AWS part.

Reply:
kk now I reflect why only smapi-command is triggered in the github action setup.